recipe/build.sh

```diff
@@ -9,8 +9,9 @@
 #
 source activate "${CONDA_DEFAULT_ENV}"
 
+autoreconf --force --install
 ./autogen.sh
-./configure --with-pic --prefix=$PREFIX
+CXXFLAGS="${CXXFLAGS} -O1 -DNDEBUG" ./configure --with-pic --prefix=$PREFIX --disable-dependency-tracking
```

Comment:
Do you want to do `-O2` on Linux? You can do in bash

`if [ "$(uname)" == "Linux" ]`